### PR TITLE
Update tutorial for OpenShift Pipelines v1.1

### PR DESCRIPTION
**Pull request by khrm**
Remove the git-repo resource and leverage git-clone task
and workspace instead. PVC has been used as workspace.
Image resource has also been removed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,32 +172,32 @@ We will be using `buildah` clusterTasks, which gets installed along with Operato
 $ tkn clustertasks ls
 NAME                       DESCRIPTION   AGE
 buildah                                  1 day ago
-buildah-v0-11-3                          1 day ago
+buildah-v0-14-3                          1 day ago
```
Comment (ppitonak) [inline README.md:175]: All those with `-pr` are missing. What if we listed only first 5 or 10 here. We just want to illustrate tha `buildah` cluster task is installed on cluster.

File: README.md [hunk 2]
```diff
@@ -37,7 +37,6 @@ Tekton defines a number of [Kubernetes custom resources](https://kubernetes.io/d
 The custom resources needed to define a pipeline are listed below:
 * `Task`: a reusable, loosely coupled number of steps that perform a specific task (e.g. building a container image)
 * `Pipeline`: the definition of the pipeline and the `Tasks` that it should perform
-* `PipelineResource`: inputs (e.g. git repository) and outputs (e.g. image registry) to and out of a pipeline or task
```
Comment (savitaashture) [inline README.md:40]: As we removed `PipelineResource` and used `Workspace` i feel its good to provide the info about workspace bit.

Comment (khrm) [reply README.md:40]: I specified that down below. Here we are only mentioning CRs.

File: README.md (hunk 1 above)
Comment (khrm) [reply README.md:175]: Yes, write. Will reduce this. We just want to illustrate.

File: README.md [hunk 3]
```diff
@@ -422,9 +355,11 @@ A `PipelineRun` is how you can start a pipeline and tie it to the git and image
...
 ```bash
 $ tkn pipeline start build-and-deploy \
-    -r git-repo=api-repo \
-    -r image=api-image \
-    -p deployment-name=vote-api
+    -w name=shared-workspace,claimName=source-pvc \
+    -p deployment-name=vote-api \
+    -p git-url=http://github.com/openshift-pipelines/vote-api.git \
+    -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-api \
+    --showlog=true
```
Comment (VeereshAradhya) [inline README.md]: If `--showlog` flag is provided, the lines  `In order to track the pipelinerun progress run:
tkn pipelinerun logs build-and-deploy-run-z2rz8 -f -n pipelines-tutorial` will not be printed

File: README.md [hunk 4]
```diff
@@ -434,9 +369,11 @@ tkn pipelinerun logs build-and-deploy-run-z2rz8 -f -n pipelines-tutorial
...
 ```bash
 $ tkn pipeline start build-and-deploy \
-    -r git-repo=ui-repo \
-    -r image=ui-image \
-    -p deployment-name=vote-ui
+    -w name=shared-workspace,claimName=source-pvc \
+    -p deployment-name=vote-api \
+    -p git-url=http://github.com/openshift-pipelines/vote-ui.git \
+    -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-ui \
+    --showlog=true
```
Comment (VeereshAradhya) [inline README.md]: same here, If --showlog flag is provided, the lines `In order to track the pipelinerun progress run:
tkn pipelinerun logs build-and-deploy-run-xy7rw -f -n pipelines-tutorial` will not be printed

**Review by vdemeester — APPROVED** [review]
/lgtm

File: README.md [hunk 5]
```diff
@@ -151,7 +144,7 @@ spec:
     - install
 ```
 
-When a task starts running, it starts a pod and runs each step sequentially in a separate container on the same pod. This task happens to have a single step, but tasks can have multiple steps, and, since they run within the same pod, they have access to the same volumes in order to cache files, access configmaps, secrets, etc. As mentioned previously, tasks can receive inputs (e.g. a git repository) and produce outputs (e.g. an image in a registry).
+When a task starts running, it starts a pod and runs each step sequentially in a separate container on the same pod. This task happens to have a single step, but tasks can have multiple steps, and, since they run within the same pod, they have access to the same volumes in order to cache files, access configmaps, secrets, etc. You can specify volume using workspace. It is recommended that Tasks uses at most one writeable Workspace. Workspace can be secret, pvc, config or emptyDir.
```
Comment (Preeticp) [inline README.md:147]: You can specify the volume using...
...that Tasks use at most...
You can specify Workspaces using secret, pvc, config or emptyDir.

File: README.md [hunk 6]
```diff
@@ -304,6 +297,8 @@ the next section.
 
 The execution order of task is determined by dependencies that are defined between the tasks via inputs and outputs as well as explicit orders that are defined via `runAfter`.
 
+`workspaces` field allow you to specify one or more volumes that each Task in the Pipeline requires during execution. You specify one or more Workspaces in the `workspaces` field.
```
Comment (Preeticp) [inline README.md:300]: The `workspaces` field allows

**Comment by Preeticp** [comment]
@khrm some minor nits, otherwise LGTM

File: README.md (hunk 6 above)
Comment (khrm) [reply README.md:300]: OK. It was already merged. Will create a new pr for this.

File: README.md (hunk 5 above)
Comment (khrm) [reply README.md:147]: I think we were talking of pod and volume in the sentence before. So specifying volume using workspace also makes sense. What do you think?

**Pull request merged**